packages/e2e/puppeteer/tests/location-component.ts

```diff
@@ -24,9 +24,12 @@ export const randomAddress = random(addresses);
 if (!randomAddress) {
   throw new Error("address not found");
 }
-const testFlow = async (page: puppeteer.Page) => {
-  await page.waitForSelector('[data-testid="next-button"]');
-  await page.click('[data-testid="next-button"]');
+const testFlow = async (page: puppeteer.Page, options: FlowOptions) => {
+  // IMTR Flow has an Intro page
```

Comment:
This comment is not fitting for the next code block.
This comment is true. However the following block is testing the restrictions under location form right?

Reply:
Well... the main difference between the two flows is that the Intro has been removed from the OLO flow. In order to test the location component the IMTR flow has to click the next button on the Intro page. That's what this is about. I will update the comment to make it more understanding